reccmp/ghidra_scripts/import_functions_and_types_from_pdb.py

```diff
@@ -70,6 +104,13 @@ def setup_logging():
 # This script can be run both from Ghidra and as a standalone.
 # In the latter case, only the PDB parser will be used.
 setup_logging()
+find_and_add_venv_to_pythonpath()
+reload_module("reccmp.ghidra_scripts.lego_util.statistics")
+reload_module("reccmp.ghidra_scripts.lego_util.globals")
+from reccmp.ghidra_scripts.lego_util.globals import GLOBALS
+
+logging.root.setLevel(GLOBALS.loglevel)
```

Comment:
This had to be reordered because we can't import `GLOBALS` before setting up the venv now. On the other hand, `mypy` doesn't allow me to import `from lego_util.globals` because the package can be referenced by two different paths.